DefaultConfigsExtension/src/test/java/de/viadee/xai/anchor/adapter/tabular/discretizer/impl/MDLPDiscretizerTest.java

```diff
@@ -0,0 +1,305 @@
+package de.viadee.xai.anchor.adapter.tabular.discretizer.impl;
+
+import de.viadee.xai.anchor.adapter.tabular.discretizer.DiscretizationTransition;
+import de.viadee.xai.anchor.adapter.tabular.discretizer.NumericDiscretizationOrigin;
+import org.junit.jupiter.api.Test;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MDLPDiscretizerTest {
+
+    @Test
+    void testMDLPBasicDiscretization(){
```

Comment:
Let's add a test case with _only_ negative values. ;)

Reply:
Added a test and the discretization works as expected.